pkg/reconciler/runner.go

```diff
@@ -33,6 +33,7 @@ type Runner struct {
 	Logger      logr.Logger
 }
 
+// TODO: make sure that model server container (deployment) is ready before creating kepler daemonset
```

Comment:
we don't need to do that ... we just have to indicate in the status that kepler `Available = false` until all resources are available.

Reply:
Could you provide me a specific guide on this or can I leave this work for the other contributor who can quickly add it? I will move todo line to another place such as line 351 in kepler.go.